src/mnemonics/electrum-words.cpp

```diff
@@ -335,6 +335,7 @@ namespace crypto
           return false;
         }
 
+        w[0] = SWAP32LE(w[0]);
```

Comment:
What's about:
```c++
dst.append(SWAP32LE((const char*)&w[0]), 4);
```

Reply:
I'm not sure what you're asking here. My original was this complicated, but w is already an array of uint32_t so I removed the casts. I also reused w since it's wiped after the fact. So it now looks much simpler and does (I think and hope) the same thing. Does that answer the question ? :)

Reply:
Both ways are fine, just `w[0] = SWAP32LE(w[0]);` will be slightly slower (under some circumstances). I asked whether you would like to optimize it a bit.

Reply:
Here, no. It's not like it'll change anything.